WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/IssueRefundViewModel.swift

```diff
@@ -147,7 +147,13 @@ final class IssueRefundViewModel {
         self.stores = stores
         self.storage = storage
         let items = refundableOrderItemsDeterminer.determineRefundableOrderItems(from: order, with: refunds)
-        state = State(order: order, refunds: refunds, itemsToRefund: items, currencySettings: currencySettings, charge: nil)
+        let thereIsOnlyCustomAmountsToRefund = items.isEmpty && order.fees.isNotEmpty
```

Comment:
Should we move this logic to some new method or property in the `OrderRefundsOptionsDeterminer`? Since already encapsulates the rest of the logic related to refunding options

Reply:
Indeed, that looks like a good place to have it. Done in 785b01492cddf55d7d0087630c97834238872d9d and 4675aaf8efe8845d59bd80ca577a253e3ae6c66f